src/libsaml.ts

```diff
@@ -263,11 +263,13 @@ const libSaml = () => {
     * @param  {array} tagValues    tag values
     * @return {string}
     */
-    replaceTagsByValue(rawXML: string, tagValues: any): string {
+    replaceTagsByValue(rawXML: string, tagValues: Record<string, unknown>): string {
       Object.keys(tagValues).forEach(t => {
+        let tagValue = tagValues[t];
+        tagValue = tagValue == null ? tagValue : tagValue.toString();
```

Comment:
If value is `null` or `undefined` leave as is (this will produce an empty string), otherwise convert to string (fixes issue with `false` values). This fixes failure happening for tests:
- flow > create login request with redirect binding using default template and parse it
- flow > create login request with post simpleSign binding using default template and parse it
- flow > create login request with post binding using default template and parse it
```
t.is(extract.nameIDPolicy.allowCreate, 'false');

  Difference:

  - ''
  + 'false'
```